Documentation/clustering.md

```diff
@@ -70,7 +70,7 @@ $ etcd -name infra2 -initial-advertise-peer-urls http://10.0.1.12:2380 \
   -initial-cluster-state new
 ```
 
-The command line parameters starting with `-initial-cluster` will be ignored on subsequent runs of etcd. You are free to remove the environment variables or command line flags after the initial bootstrap process. If you need to make changes to the configuration later (for example, adding or removing members to/from the cluster), see the [runtime configuration](runtime-configuration.md) guide.
+The command line parameters starting with `-initial-cluster` will be ignored on subsequent runs of etcd. You are free to remove the environment variables or command line flags after the initial bootstrap process. If you need to make changes to the configuration later (for example, adding or removing members to/from the cluster), see the [runtime configuration][runtime-conf] guide.
```

Comment:
Both work, but i'm not sure which way is preferred. `--help` flag prints options with the double-dash. So, let's use double dash.